### ISSUE TITLE
removeCallback feature

### ISSUE THREAD

**Issue by wizardsambolton** [TalAter/annyang#112]
is there anyway to reset callbacks to default?

the callback being the noresults callback

**Comment by MartinKoutny** [comment]
+1

**Comment by TalAter** [comment]
This feature was added in the just released [v2.2.1](https://github.com/TalAter/annyang/releases/tag/v2.2.1).

What do you think?

## removeCallback(type, callback)

Remove callbacks from events.

- Pass an event name and a callback command to remove that callback command from that event type.
- Pass just an event name to remove all callback commands from that event type.
- Pass undefined as event name and a callback command to remove that callback command from all event types.
- Pass no params to remove all callback commands from all event types.

#### Examples:
````javascript
annyang.addCallback('start', myFunction1);
annyang.addCallback('start', myFunction2);
annyang.addCallback('end', myFunction1);
annyang.addCallback('end', myFunction2);

// Remove all callbacks from all events:
annyang.removeCallback();

// Remove all callbacks attached to end event:
annyang.removeCallback('end');

// Remove myFunction2 from being called on start:
annyang.removeCallback('start', myFunction2);

// Remove myFunction1 from being called on all events:
annyang.removeCallback(undefined, myFunction1);
````
